dev-libs/libphonenumber/libphonenumber-8.12.26.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=emake
+
+inherit cmake java-pkg-2
+
+DESCRIPTION="library for parsing, formatting, and validating international phone numbers"
+HOMEPAGE="https://github.com/google/libphonenumber"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm64"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+RDEPEND="
+	dev-libs/boost
+	dev-libs/icu
+	dev-libs/protobuf
+"
+
+BDEPEND="
+	dev-cpp/gtest
```

Comment:
`dev-cpp/gtest` is C++ library for testing, so when it is used it should be in `DEPEND` and usually in `test? ( ... )` conditional.
But with `RESTRICT="test"`, maybe disable any usage of `dev-cpp/gtest` and leave `dev-cpp/gtest` only in comment.
You also need `DEPEND` anyway:
```
DEPEND="${RDEPEND}"
# test? ( dev-cpp/gtest )

Reply:
Does not build without great, that's why it is on BDEPEND (as it is required only during build time)

Reply:
We should make the gtest dependency optional. DEPEND is also build-time only but it's about where the dependency is satisfied.